doc/reference/reference_lua/net_box.rst

```diff
@@ -60,25 +60,29 @@ The diagram below shows possible connection states and transitions:
 
 .. ifconfig:: builder not in ('latex', )
 
-    .. image:: net_states.svg
+    .. image:: net_states.png
         :align: center
-        :alt: net_states.svg
+        :alt: net_states.png
```

Comment:
"miro" logo on the diagram looks redundant.

Switching from a vector to raster image isn't good.

Reply:
Removed the logo (it appeared because of Miro Free).

According to the new style guide (style guide is in progress), we use Miro and .png files for such diagrams.